server/src/com/thoughtworks/go/config/GoConfigDao.java

```diff
@@ -187,9 +192,11 @@ public void updatePipeline(PipelineConfig pipelineConfig, LocalizedOperationResu
 
     public ConfigSaveState updateConfig(UpdateConfigCommand command) {
         Context context = metricsProbeService.begin(ProbeType.UPDATE_CONFIG);
+        LOGGER.info("Config update request by {} is in queue - {}", UserHelper.getUserName().getUsername(), command);
         try {
             synchronized (writeLock) {
                 try {
+                    LOGGER.info("Config update request by {} is being processed", UserHelper.getUserName().getUsername());
```

Comment:
I thought about it. I think it makes sense. It's usually a problematic activity on medium to big setups, with the administrators having no real view into it. Given that it is usually infrequent, I chose INFO. We can rethink it if we hear feedback. I doubt there will be.